.github/workflows/image.yml

```diff
@@ -0,0 +1,37 @@
+name: Publish Docker image
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    steps:
+      - name: Login to Docker Hub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKERHUB_USER }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
```

Comment:
`secrets.DOCKERHUB_TOKEN` to `secrets.DOCKERHUB_PASSWORD`

https://docs.github.com/en/actions/reference/encrypted-secrets